addons/event_sale/tests/test_event_sale.py

```diff
@@ -231,6 +231,13 @@ def test_event_crm_sale(self):
         self.assertEqual(editor_action['type'], 'ir.actions.act_window')
         self.assertEqual(editor_action['res_model'], 'registration.editor')
 
+    def test_event_sale_free_no_saleorder(self):
+        ticket = self.env['event.registration'].create({
```

Comment:
Also, this should be called "registration", as it's not a ticket, a bit confusing otherwise.

Reply:
right (y)